package.json

```diff
@@ -25,19 +25,31 @@
     },
     "author": "Frontify Developers <developers@frontify.com>",
     "dependencies": {
+        "@babel/plugin-proposal-class-properties": "^7.16.0",
+        "@babel/plugin-transform-react-jsx": "^7.16.0",
+        "@babel/preset-env": "^7.16.4",
+        "@babel/preset-react": "^7.16.0",
+        "@babel/preset-typescript": "^7.16.0",
         "@rollup/plugin-commonjs": "21.0.1",
         "@rollup/plugin-json": "4.1.0",
         "@rollup/plugin-node-resolve": "13.0.6",
         "@rollup/plugin-replace": "3.0.0",
+        "babel-loader": "^8.2.3",
+        "css-loader": "^6.5.1",
         "esbuild": "0.14.1",
         "fastify": "3.24.1",
         "fastify-cors": "6.0.2",
         "fastify-static": "4.5.0",
         "fastify-websocket": "4.0.0",
+        "minimist": "^1.2.5",
+        "minimist-options": "^4.1.0",
```

Comment:
Is it required to be in `dependencies`?
As everything is bundled it should not

Reply:
i guess it's a decision of the package scope. since this one is about to provide the tooling, it cannot run without having it, so it's a dependency imho